public/Lab-Files/Lesson03/Challenges/wild-animal.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Wild Animal</title>
+    <style>
+        main {
+            background: #8a7559;
+            text-align: center;
+            color: ivory;
+            border: 3px solid #211309;
+        }
+        
+        .image {
+            border: 2px solid #211309;
+        }
+        
+        .big {
+            color: #211309;
+        }
+    </style>
+</head>
+<body>
+<main>
+    <h1 class="big">The coolest creature</h1>
+
+        <img src="Images/elephant.jpg" alt="Lone elephant walking along dirt road" class="image"/>
+        
+        <p>The <em>elephant</em> is awesome because it is my <strong>favorite animal</strong>.</p>
+        <p>Now originally it was my favorite animal growing up because it was the only animal I could think of that started with the letter <em>E</em> (the first letter of my name). <br>
+        However, since adolescence, I have found that the elephant is a fasinating animal and still my favorite.</p>
+
+    <h3 class="big">Here are some reasons to find elephants super cool:</h3>
+    
+        <p>They are considered a <strong>keystone species</strong> due to their impact on the environment. Basically, they can make
+        or break the area they are in depending on their actions.<br> For a positive example, they dig waterholes in times of drought that 
+        are later used by other animals. On the flipside, if they tred around an area enough,<br> their massive weight will compact the 
+        dirt together making the rain run off and cause erosion.</p>
+        
+        <p>Elephants are also creatures of importance in African and Asian religions. Certain African tribes considered <br>
+        elephants to be the reincarnations of their chieftans and would bury their leaders with the ivory tusks of the elephant.<br>
+        In Asian religions such as Hindu, the god <em>Airavata</em>, is the father of elephants and controls rainbows and lightning.<br>
+        Another Hindu god, <em>Ganesha</em>, often depicted with an elephant-like head, is believed to be the giver of success and desires.<br>
+        In Buddhism, it is believed the <strong>Buddha</strong> himself was reincarnated from a white elephant.</p>
+        
+</main>    
+</body>
+</html>
```

Comment:
Erin, you did great with all of these challenges. You are doing an awesome job organizing your code too. Keep up the good work!